tasks/publish.js

```diff
@@ -1,5 +1,6 @@
 "use strict"
 
+const build = require(`./build/build`)
```

Comment:
Oops, I had already fixed that myself but it looks like my push failed.  Thanks for catching it.